structuralcodes/material/concrete/__init__.py

```diff
@@ -0,0 +1,58 @@
+"""Concrete material"""
```

Comment:
Note that this file is now in structuralcodes/material/concrete. Please make sure that you rename your folder accordingly, and the diff disappears.